sklearn/preprocessing/_data.py

```diff
@@ -3205,6 +3205,18 @@ def power_transform(X, method='yeo-johnson', *, standardize=True, copy=True):
 
     Read more in the :ref:`User Guide <preprocessing_transformer>`.
 
+    .. warning:: Risk of data leak
```

Comment:
Is there an agreement on where this section should be placed ?
I would put it further below, before the section "See also".

Reply:
I'd prefer having it here so that it's immediately visible

@abbiepopa thanks for the PR, it seems that you'll need to indent that block to the right

Reply:
ok

Reply:
Indented block; rerunning checks now